Makefile

```diff
@@ -136,9 +136,15 @@ neon-pg-ext-%: postgres-%
 
 .PHONY: neon-pg-ext-clean-%
 neon-pg-ext-clean-%:
-	$(MAKE) -C $(POSTGRES_INSTALL_DIR)/pgxn/neon-$* -f $(ROOT_PROJECT_DIR)/pgxn/neon/Makefile clean
-	$(MAKE) -C $(POSTGRES_INSTALL_DIR)/pgxn/neon_walredo-$* -f $(ROOT_PROJECT_DIR)/pgxn/neon_walredo/Makefile clean
-	$(MAKE) -C $(POSTGRES_INSTALL_DIR)/pgxn/neon_test_utils-$* -f $(ROOT_PROJECT_DIR)/pgxn/neon_test_utils/Makefile clean
+	$(MAKE) PG_CONFIG=$(POSTGRES_INSTALL_DIR)/$*/bin/pg_config \
```

Comment:
why do you need this?

Reply:
Because `pgxn` makefile needs it to figure out path to the right version

Reply:
it makes sense only for local development, because when we building it in Docker - the step is different

Reply:
or, I just saw your comment above

```
I occasionally pushed Makefile commit into this PR as well.

It's trivial, so I won't bother splitting it into another PR
```

Understood then. I tried to match these changes with the main story